Build failure with J9ClassEnv.cpp

Two calls to `inROMClass()` in `J9::ClassEnv::getROMClassRefName()` cause a build failure when you enable `TR_ASSERT()` as shown below:

>/root/openj9-openjdk-jdk11/build/linux-aarch64-normal-server-release/vm/compiler/../compiler/env/J9ClassEnv.cpp: In member function 'uint8_t* J9::ClassEnv::getROMClassRefName(TR::Compilation*, TR_OpaqueClassBlock*, uint32_t, int&)':
>/root/openj9-openjdk-jdk11/build/linux-aarch64-normal-server-release/vm/compiler/../compiler/env/J9ClassEnv.cpp:520:14: error: 'inROMClass' was not declared in this scope
>    TR_ASSERT(inROMClass(romFieldRef), "field ref must be in ROM class");

--- a/runtime/compiler/env/J9ClassEnv.cpp
+++ b/runtime/compiler/env/J9ClassEnv.cpp
@@ -517,9 +517,7 @@ J9::ClassEnv::getROMClassRefName(TR::Compilation *comp, TR_OpaqueClassBlock *cla
    {
    J9ROMConstantPoolItem *romCP = self()->getROMConstantPool(comp, clazz);
    J9ROMFieldRef *romFieldRef = (J9ROMFieldRef *)&romCP[cpIndex];
-   TR_ASSERT(inROMClass(romFieldRef), "field ref must be in ROM class");
    J9ROMClassRef *romClassRef = (J9ROMClassRef *)&romCP[romFieldRef->classRefCPIndex];
-   TR_ASSERT(inROMClass(romClassRef), "class ref must be in ROM class");
 #if defined(JITSERVER_SUPPORT)
    if (comp->isOutOfProcessCompilation())
       {

